modules/s3/src/main/scala/com/rewardsnetwork/pureaws/s3/S3ObjectInfo.scala

```diff
@@ -1,9 +1,9 @@
 package com.rewardsnetwork.pureaws.s3
 
-import java.time.Instant
-
 import software.amazon.awssdk.services.s3.model.S3Object
 
+import java.time.Instant
```

Comment:
Is there a reason this was moved down? Not that it matters but it seems like it might have been intentional, or the result of some kind of code formatting.

Reply:
That's definitely `scalafmt`, our defaults are to put native language packages `java` and `scala` at the bottom of the import list. If this is an issue I can put a new commit up with it restored.

Reply:
Yes if you can move it to where it was previously, that'd be perfect. Otherwise I think this looks good!

Reply:
Done